source/MRViewer/MRImGuiImage.h

```diff
@@ -140,6 +140,11 @@ class MarkedVoxelSlice : public ImGuiImage
     // Set current slice with marks to texture, do not abuse this
     MRVIEWER_API void forceUpdate();
 
+    //save the slice by the active plane through the current voxel to an image file
+    MRVIEWER_API void saveCurrentSliceToFile( const std::filesystem::path& path );
+    //save all slices by the active plane through all voxels along the active axis to an image file
+    MRVIEWER_API void saveAllSlicesToDirectory( const std::filesystem::path& path );
```

Comment:
better have separate functions, not part of ImGuiImage (I think we need it in MRMesh project)